From 3e1ccdb5584c84a60815c207c9abaadba093e54e Mon Sep 17 00:00:00 2001
From: tonishi92 <tatsuo.onishi@gmail.com>
Date: Tue, 6 Aug 2024 12:47:55 +0200
Subject: [PATCH 48/55] Remove call to zero_out for chem

In solve_em, remove a call to microphysics_zero_outb so that small chem
values are not reset to mp_zero_out_thresh. The default value of
mp_zero_out_thresh is 1.0E-8, but some trace chem species and radicals
should be allowed to be smaller than this.
---
 dyn_em/solve_em.F | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/dyn_em/solve_em.F b/dyn_em/solve_em.F
index c5f47a50..ab753503 100644
--- a/dyn_em/solve_em.F
+++ b/dyn_em/solve_em.F
@@ -4022,12 +4022,6 @@ BENCH_START(moist_phys_end_tim)
                       its, ite, jts, jte,                               &
                       k_start    , k_end                                )
 
-       CALL microphysics_zero_outb (                                    &
-                      chem , num_chem , config_flags ,                  &
-                      ids, ide, jds, jde, kds, kde,                     &
-                      ims, ime, jms, jme, kms, kme,                     &
-                      its, ite, jts, jte,                               &
-                      k_start    , k_end                                )
        CALL microphysics_zero_outb (                                    &
                       tracer , num_tracer , config_flags ,              &
                       ids, ide, jds, jde, kds, kde,                     &
-- 
2.31.1

